src/arena/ArenaDashboard.js

```diff
@@ -1,9 +1,10 @@
 import React, { useState } from "react";
 import Image from "react-bootstrap/lib/Image";
+import { useTheme } from "@mui/material";
 import { I18n } from "@aws-amplify/core";
 import Tour from "reactour";
 import ArenaDashboard from "./Sprints";
-import question from "../assets/help.png";
+import question from "../assets/question.svg";
```

Comment:
Love replacing the png with an SVG. As an FYI, this is an awesome resource for finding amazing SVG's.

Reply:
https://www.svgrepo.com

Reply:
Nice! Yeah, I just grabbed one quickly from Bootstrap, would love to open a ticket for someone more design-oriented to standardize all the SVGs and provide from creative direction for contributors sourcing ones in the future.